test/e2e/elasticsearch_test.go

```diff
@@ -27,13 +28,13 @@ type ElasticSearchTestSuite struct {
 	suite.Suite
 }
 
-var esEnabled = false
+var esIndexCleanerEnabled = false
```

Comment:
I am not sure how this got here but I would recommend removing global variables in future